### PR TITLE
Added some simple validation. JS isn't perfect.

### PR DESCRIPTION
**Pull request by rlsaylor**
Got confused with the angular validation methods. I want the "error" form to only appear when the user clicks in the field ($dirty) and the input is not valid ($invalid). For some reason, the error messages are always there. Probably need to do something in configApp.js

### CODE REVIEW

File: src/main/webapp/WEB-INF/tiles/layout/config/body.jsp [hunk 1]
```diff
@@ -12,13 +12,20 @@
                             <div class="col-xs-5">
                                 <div class="form-group">
                                     <label>Attribute</label>
```
Comment (crstalli) [inline src/main/webapp/WEB-INF/tiles/layout/config/body.jsp]: Label needs a for=

Comment (crstalli) [inline src/main/webapp/WEB-INF/tiles/layout/config/body.jsp]: Label needs a for=

Comment (crstalli) [inline src/main/webapp/WEB-INF/tiles/layout/config/body.jsp]: You don't want to hard code  the validation messages individually

**Comment by crstalli** [comment]
Sort of-- Also, you never want to do inline styles unless you can help it. Utelize bootstrap pre-made styles when you can. I can show you if you want. Try to just get the default angular validation working. Make sure there are no spaces in the fields, and that they are not blank before the add button is clicked.

**Comment by rlsaylor** [comment]
Ok I'll keep working on it tonight, and try to push something tomorrow.

**Comment by crstalli** [comment]
This is a good demo. I am not used to angular validation but it seems like they do it like you did it.

https://scotch.io/tutorials/angularjs-form-validation#demo

This uses bootstrap validation,.

**Pull request closed without merging**